nautobot_ssot_infoblox/diffsync/adapters/nautobot.py

```diff
@@ -97,12 +111,33 @@ def __init__(self, *args, job=None, sync=None, **kwargs):
         super().__init__(*args, **kwargs)
         self.job = job
         self.sync = sync
+        self.objects_to_create = defaultdict(list)
+
+    def sync_complete(self, source: DiffSync, *args, **kwargs):
+        """Process object creations/updates using bulk operations.
+
+        Args:
+            source (DiffSync): Source DiffSync adapter data.
+        """
+        if len(self.objects_to_create["vlangroups"]) > 0:
+            self.job.log_info(message="Performing bulk create of VLAN Groups in Nautobot")
+            VLANGroup.objects.bulk_create(self.objects_to_create["vlangroups"], batch_size=250)
```

Comment:
which is the creiteria fro the batch sizes? why not use only one? and make it customizable?

Reply:
Sure, we can make it customizable. I set the sizes with the following assumptions:
1. There typically aren't a ton (>1000) VLAN Views defined in Infoblox so small batch sizes should be acceptable.
2. VLANs and Prefixes are the next biggest sets of objects that would be imported so went with double the size of VLAN Views.
3. As you'll most likely be importing thousands of IP Addresses it seemed to make sense to use that as the size for that object.

I was also trying to go with what I thought was a good balance between not slamming the DB hard with data and still making large imports occur in a reasonable amount of time, ie not hours.